server/chat-plugins/poll.ts

```diff
@@ -350,6 +350,13 @@ export const commands: ChatCommands = {
 
 		clearqueue: 'deletequeue',
 		deletequeue(target, room, user, connection, cmd) {
+			const [slotString, roomid, update] = target.split(',').map(i => i.trim());
+			if (roomid) {
+				const targetRoom = roomid ? Rooms.search(roomid) : room;
+				if (targetRoom) {
+					this.room = targetRoom;
```

Comment:
Wouldn't setting `this.room` make it difficult to tell if the command succeeded or failed, since its output would be in a different room than it was used in? Or is this command only used from the poll management page? (I dislike using pages personally, and prefer commands. I don't know if others share this preference.)

Reply:
This is meant for the page, yes.

Reply:
`this.room` has to be set for it to work.

Reply:
I'm pretty sure `this.room` does not actually need ot be set for it to work.